site/scripts/library.js

```diff
@@ -404,6 +404,50 @@ function showMangaDetails(manga, categories, source) {
     }
   })();
 
+  ///////////////////
+  const trackingImages = [
+    { src: 'img/trackers/ic_tracker_mal.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_anilist.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_kitsu.webp', greyed: true },
+    { src: 'img/trackers/ic_manga_updates.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_shikimori.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_bangumi.webp', greyed: true },
+  ];
+
+  // Extract sync IDs if manga.tracking exists
+  const syncIds = manga.tracking
+    ?.map((track, index) => ({ syncId: track?.syncId, trackingUrl: track?.trackingUrl, index }))
+    .filter(item => item.syncId != null);
+
+  // Update trackingImages based on syncIds
+  syncIds?.forEach(item => {
+    if (item.syncId >= 1 && item.syncId <= 6) {
+      trackingImages[item.syncId - 1].greyed = false;
+      trackingImages[item.syncId - 1].trackingUrl = item.trackingUrl;
+      trackingImages[item.syncId - 1].index = item.index;
+    }
+  });
+
+  const mangaTracking = document.getElementById('manga-tracking');
+  mangaTracking.innerHTML = '';
+
+  trackingImages.forEach((item, index) => {
+    const li = document.createElement('li');
+    li.style.backgroundImage = `url(${item.src})`;
+    if (item.greyed) {
+      li.classList.add('manga-tracking-greyed');
+    } else {
+      li.addEventListener('click', () => {
+        if (item.trackingUrl) {
+          window.open(item.trackingUrl, '_blank');
+        }
```

Comment:
Use a `<a>` instead of click event.
Can't copy the URL from that nor check it on hover